N/NCCL/build_tarballs.jl

```diff
@@ -1,48 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
 using BinaryBuilder, Pkg
-using Base.BinaryPlatforms: arch, os
 
 const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "NCCL"
-version = v"2.18.5"
-build = 1
+version = v"2.19.4"
 
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/NVIDIA/nccl.git", "88d44d777f6970bdbf6610badcbd7e25a05380f0"),
+]
+
+# Bash recipe for building across all platforms
 script = raw"""
-mkdir -p ${libdir} ${prefix}/include
+cd $WORKSPACE/srcdir
 
-cd ${WORKSPACE}/srcdir/nccl*
-find .
+export TMPDIR=${WORKSPACE}/tmpdir
+export CUDA_HOME=${WORKSPACE}/destdir/cuda
+export CUDA_LIB=${CUDA_HOME}/lib
+export CXXFLAGS='-D__STDC_FORMAT_MACROS'
+export CUDARTLIB=cudart
 
-install_license LICENSE.txt
```

Comment:
We still need to keep the license in the build

Reply:
It's already in the tarball: is that sufficient?
```
├── LICENSE.txt
├── include
│   ├── nccl.h
│   └── nccl_net.h
├── lib
│   ├── libnccl.so -> libnccl.so.2
│   ├── libnccl.so.2 -> libnccl.so.2.19.4
│   ├── libnccl.so.2.19.4
│   ├── libnccl_static.a
│   └── pkgconfig
│       └── nccl.pc
└── share
    └── licenses
        └── NCCL
            └── LICENSE.txt
```